src/graphics/vulkan/vulkan_texture.cpp

```
@@ -281,6 +281,51 @@ namespace moth_graphics::graphics::vulkan {
 
+    void Texture::UpdatePixels(IntRect const& destRect, uint8_t const* pixels) {
+        if (pixels == nullptr) {
+            return;
+        }
+        auto const regionWidth = static_cast<uint32_t>(destRect.w());
+        auto const regionHeight = static_cast<uint32_t>(destRect.h());
+        if (regionWidth == 0 || regionHeight == 0) {
+            return;
+        }
+
+        VkDeviceSize const regionSize = static_cast<VkDeviceSize>(regionWidth) * regionHeight * 4;
```

Clang (14.0.6)

[error] variable 'regionSize' is not initialized

(cppcoreguidelines-init-variables,-warnings-as-errors)